backends-common/cassandra/src/test/java/org/apache/james/backends/cassandra/DockerCassandraRule.java

```diff
@@ -145,4 +145,12 @@ public void unpause() {
         client.unpauseContainerCmd(cassandraContainer.getContainerId());
     }
 
+    public TestRule getLifecycleTestRule(int iterationsBetweenRestart) {
```

Comment:
Can you add a `defaultLifecycleTestRule` method in this class, then use it in tests.
This method must use a constant to define the number of iterations, that way all test classes will have the same number of iterations.

Another thing, is that I'm in favor to reduce this number in order to have a really really safe build, let say may be 10.

Reply:
> Another thing, is that I'm in favor to reduce this number in order to have a really really safe build, let say may be 10.

I don't agree : it makes things slower, we should keep a value close enough to the limit.

Reply:
Then, how do we calculate this?

Reply:
bisect. The good thing is that we already did it, we just have to keep the current value.

Reply:
Yes, it was on the server called _yoda_ which has 32GB.
But know we are also building on _anakin_ which has 16GB.

Reply:
It doesn't depend on the host because we require a fix amount of memory at container start.

Reply:
Yes it depends, because some of them already have lot of stuff.
What I'm looking first is not a fast build, but a build in which I can trust.
Actually, there are too much random/unexpected failure, then I can't trust in the build any more.
We have to reach a stable build, then I agree with you to enhanced it.